x-pack/elastic-agent/pkg/capabilities/testdata/allow_metrics-capabilities.yml

```diff
@@ -0,0 +1,4 @@
+- rule: allow
```

Comment:
Having a top level entry `capabilities` instead of directly the array would allow use to extend it later with other things if needed. For example the version of the capabilities file.

Reply:
+1 on adding a version from day 1.